### DISCUSSION TITLE
Welcome to UVR Discussions!

### DISCUSSION THREAD

**Discussion by Anjok07** [Anjok07/ultimatevocalremovergui#424] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.
  * Be respectful and kind to one another.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hatrd** [comment]
Greetings, I wonder how's the chinese interface coming along? I just saw it needs more test to release, and I am willing to help!

**Comment by gratesmanes** [comment]
nice , i wonder if there's a remove backing vocals model only to just have the main voice.

**Reply by Karamel69** [reply]
I try to find the same with different word, how to separate two vocals when they are mixed...
thank you in advance

**Comment by davo2525** [comment]
Hello, trying to get a specific result from UVR. I'm trying to separate the guitar from the keyboard/synth sounds. I've been able to separate the guitar and synth from the vocal/bass/drum, but the guitar and synth are together as "other". Perhaps someone has a suggestion. Thanks

**Comment by bumper2148** [comment]
greetings everyone , hope i can do something with this, not very smart, lol

**Comment by countryritchie** [comment]
hi.can anyone give me the items i need to use for extracting just an instrumental (removing lead vocal) and also extracting instrumental with backing vocals please ?

**Comment by JeffersonStarseed** [comment]
Wanted to say that I downloaded UVR5 and tested it out.  I have used different products to attempt to make backing tracks or just instrumental rhythm section tracks without the vocals for a while, since 2016.  Even recently, I am a user of Virtual DJ, which has a pretty good stems feature built-in since they upgraded it to it's 2.0 recently.  However, and that being said, UVR5 blows everything else away.  When it comes to making basic instrumental tracks and removing vocals with ease, UVR5 does it with the least-compromising of "quality" of the remaining leftover tracks/media (compared to all the others).  Kudos, and I really like what you have done here.

**Reply by gratesmanes** [reply]
This is very true i discovered also removing crowds and applauding in concerts , you end up with perfect song and voice of the singers .
Nothing surpasses this UVR software in my view best karaoke , instrumentals or acapella

**Reply by Bobo-Hui647** [reply]
I am looking for the same solution.  Would you share which model you were using to remove the crowds and applaud in concerts?
Thanks

**Comment by Sooz55** [comment]
Hi! Total newb here...unfortunately I have Catalina 10.15.7 on a late 2013. 
Is there ANY hope of installing an older version?

**Reply by zdd123456M** [reply]
When will the Ultimate Vocal Remover UVR5 be updated

**Comment by vocal-star** [comment]
What UVR model extracts backing vocals?
